modules/database/src/main/java/org/eclipse/xpanse/modules/database/servicetemplate/ServiceTemplateEntity.java

```diff
@@ -69,4 +69,10 @@ public class ServiceTemplateEntity extends CreateModifiedTime {
     @Column(name = "SERVICE_REGISTRATION_STATE")
     @Enumerated(EnumType.STRING)
     private ServiceRegistrationState serviceRegistrationState;
+
+    @Column(name = "JSON_OBJECT_SCHEMA_StRING", columnDefinition = "json", nullable = false)
+    @Type(value = JsonType.class)
+    @Convert(converter = ObjectJsonConverter.class)
+    private String jsonObjectSchemaString;
```

Comment:
please see if we can use JsonObjectSchema type instead of string. 
Also see in the h2-console, how the data is stored. Should be same as OCL column.